channel/consensus_channel/consensus_channel.go

```diff
@@ -34,10 +34,11 @@ type consensusChannel struct {
 func newConsensusChannel(
 	fp state.FixedPart,
 	myIndex ledgerIndex,
+	turnNum uint64,
```

Comment:
Would `initialTurnNum` be more clear?

Reply:
Updated in [253b246](https://github.com/statechannels/go-nitro/pull/433/commits/253b246559ca0980e34344e75a4a4c928a4f332b)